diff --git a/samples/CMakeLists.txt b/samples/CMakeLists.txt
index 34caf483..97d27c49 100644
--- a/samples/CMakeLists.txt
+++ b/samples/CMakeLists.txt
@@ -103,3 +103,23 @@ if (CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE)
     DOMAIN CPUAPP
     BOARD ${CONFIG_DOMAIN_CPUAPP_BOARD})
 endif()
+
+# Special handling for the nRF53 when using 802.15.4 serialization.
+# Automatically include the 802154_rpmsg sample as child image, and change
+# the board to be the network core.
+if (CONFIG_NRF_802154_SER_HOST)
+  if (CONFIG_SOC_NRF5340_CPUAPP)
+
+    message("Adding '802154_rpmsg' sample as child image since "
+      "CONFIG_NRF_802154_SER_HOST is set to 'y'")
+
+    add_child_image(
+      NAME 802154_rpmsg
+      SOURCE_DIR ${ZEPHYR_BASE}/samples/boards/nrf/ieee802154/802154_rpmsg
+      DOMAIN CPUNET
+      BOARD ${CONFIG_DOMAIN_CPUNET_BOARD})
+  else()
+    message(FATAL_ERROR
+      "SoC not supported for including 802154_rpmsg as child image")
+  endif()
+endif()
